share/templates/CMakeLists.txt

```diff
@@ -17,20 +17,53 @@
 #  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 #=============================================================================
 
+# Usage: install_templates(install_dir [mscx_templates...])
+# MSCX templates paths should be relative to CMAKE_CURRENT_SOURCE_DIR.
+function(install_templates DESTINATION_DIR)
+      # Create a directory for the converted templates
+      get_filename_component(MSCZ_BUILD_DIR "${ARGV1}" DIRECTORY)
+      file(MAKE_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/${MSCZ_BUILD_DIR}")
+
+      # Remove DESTINATION_DIR from the list
+      list(REMOVE_AT ARGV 0)
```

Comment:
FYI there is another list variable `ARGN` that will have DESTINATION_DIR already removed. See [CMake function](https://cmake.org/cmake/help/v3.0/command/function.html).

Reply:
I didn't notice the note about `ARGN` there, thanks!